transcendence/api.py

```diff
@@ -1,8 +1,6 @@
-import jwt
 import requests
 import shortuuid
-from . import user
-from django.conf import settings
+from .user import *
```

Comment:
i was trying to fix something, and this make it hard to debug. better to be clear what function comes from what module